README.adoc

```diff
@@ -78,15 +78,14 @@ and what it can access by providing it with a simple collection of links to othe
 
 === Response JSON
 
-When you build a RESTful web service, it is important to consider the style of your response files.
+When you build a RESTful web service, consider the style of your response files.
 Whether they are JSON files, XML files, or in some other format, a good practice is to always have them in
 a clean and organized form. In the context of HATEOAS, each resource must contain a link reference to
-itself, commonly referred to as `self`. Each link must also have a relationship associated with it,
-although no strict rules exist as to how this relationship must be formatted. The collection
-of such links must be contained with a `_links` array, which itself must be a direct property of the
-resource object. The underscore is used so that the `_links` property does not collide with any
-existing fields named `links`. The structure of HATEOAS links that you will be using in this
-guide is the following one:
+itself, commonly referred to as `self`. Each link also needs a relationship to be associated with it,
+although no strict rules exist as to how you need to format this relationship. Contain the collection
+of such links with a `_links` array, which itself must be a direct property of the
```

Comment:
I like the change, but perhaps we can instead say "within a '_links' array"?

Reply:
Yes, now that you mention it, I agree that "within" is better than 'with." :)